spec/influxdb/cases/query_cluster_spec.rb

```diff
@@ -17,7 +17,7 @@
   describe "#create_cluster_admin" do
     let(:user) { 'adminadmin' }
     let(:pass) { 'passpass' }
-    let(:query) { "CREATE USER #{user} WITH PASSWORD '#{pass}' WITH ALL PRIVILEGES" }
+    let(:query) { "CREATE USER \"#{user}\" WITH PASSWORD '#{pass}' WITH ALL PRIVILEGES" }
```

Comment:
Why is pass in single quotes and user in double quotes?

Reply:
Because that's what we generate now. See #248